src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -104,13 +107,46 @@ const keriAcdcChangeHandler = async (
     dispatch(setCurrentOperation(OperationType.IDLE));
   }
 };
-
+const ACTIVTY_TIMEOUT = 14000;
```

Comment:
Any suggestions on where to put this constant?

Reply:
This is fine for me if it's an actual constant but if we want it to be configurable then it should go to the config yaml files and configurationService.ts. Would maybe be a good thing since then we can launch have different values for dev vs TestFlight.

Reply:
I would go higher than 14s too I think, maybe can see what Revolut uses. We just want to add some security for when it's been in the background for a long time, and not be overly intrusive when they are switching between apps to copy something etc.

Reply:
ok we can make it configurable later on, even from the settings page

Reply:
I set the timeout to 60 sec

Reply:
I don't think this needs to be in the settings page, to be honest